src/coreComponents/physicsSolvers/wavePropagation/WaveSolverBase.hpp

```diff
@@ -54,11 +54,28 @@ class WaveSolverBase : public SolverBase
 
     static constexpr char const * rickerOrderString() { return "rickerOrder"; }
     static constexpr char const * outputSeismoTraceString() { return "outputSeismoTrace"; }
+    static constexpr char const * dtSeismoTraceString() { return "dtSeismoTrace"; }
+    static constexpr char const * indexSeismoTraceString() { return "indexSeismoTrace"; }
+
 
   };
 
+
+
 protected:
 
+  /**
+   * @brief Re-initialize source and receivers positions in the mesh, and resize the pressureNp1_at_receivers array
+   */
+  virtual void reinit() override final;
```

Comment:
BTW, you do not need both `override` and `virtual`. You can just keep `override`.

Reply:
Done.